app/assets/javascripts/discourse/app/lib/render-topic-featured-link.js

```diff
@@ -11,6 +11,16 @@ export function addFeaturedLinkMetaDecorator(decorator) {
 export function extractLinkMeta(topic) {
   const href = topic.get("featured_link");
   const target = User.currentProp("external_links_in_new_tab") ? "_blank" : "";
+  const domain = topic.get("featured_link_root_domain");
+  let allowList = Discourse.SiteSettings.exclude_rel_nofollow_domains;
```

Comment:
We have to move away from `Discourse.SiteSettings` constant usage as it's a global variable.

Instead could you pass in `allowList` as an option to this function? See this PR didn't add this but the same approach should be done for `User.currentProp` - we could pass that in as an option `externalLinksInNewTab: true` when calling the function.

This is necessary for newer versions of Ember and to allow us to upgrade our application.

Reply:
> See this PR didn't add this but the same approach should be done for `User.currentProp`

If I update it then I have to change 3 other files which are not related to this PR. So I will do it in a separate PR.